be/src/storage/update_manager.cpp

```diff
@@ -32,6 +32,9 @@ UpdateManager::UpdateManager(MemTracker* mem_tracker)
 }
 
 UpdateManager::~UpdateManager() {
+    if (_apply_thread_pool != nullptr) {
+        _apply_thread_pool->shutdown();
+    }
```

Comment:
Would better add some comment for these statements order.

Reply:
OK